build.gradle

```diff
@@ -13,6 +13,7 @@ group = project.maven_group
 
 loom {
 	accessWidenerPath = file("src/main/resources/carpet.accesswidener")
+	runtimeOnlyLog4j = false // Can be turned on after TriConsumer is no longer present in public API
```

Comment:
what this have to do with the TriConsumer in Setting Manager?

Reply:
Oh, didn't realized that it was from log4j. I would say burn it with fire? question when.

Reply:
Neither did I, until gradle complained.

Yeeting it right now would make any already compiled extension that uses rule observers crash on launch, I think it's fine to wait a bit while it's deprecated and then removed.